_posts/en/newsletters/2021-01-13-newsletter.md

```diff
@@ -138,7 +138,18 @@ repo], [Hardware Wallet Interface (HWI)][hwi repo],
 
 - [Bitcoin Core #18077][] net: Add NAT-PMP port forwarding support FIXME:Xeyko
 
-- [Bitcoin Core #19055][] muhash FIXME:adamjonas
+- [Bitcoin Core #19055][] adds the Muhash algorithm and uses it to calculate
+  the UTXO set hash in `gettxoutsetinfo`.  As covered in
```

Comment:
Are you sure it gets used?  I thought this was only adding the function, with 19145 actually adding it to the RPC.

Reply:
Whoops. Updated.